hack/deploy-logging.sh

```diff
@@ -168,15 +168,24 @@ get_cluster_version_maj_min() {
     fi
 }
 
+MASTER_RELEASE_VERSION=4.6
```

Comment:
I thought we moved away from needing to hardcode this here.  Isn't there a VAR that comes from CI that tells us the release stream?  @bparees   would it make more sense to place this in the Makefile?

Reply:
i'm not aware of one....at best there might be one that tells you the branch (e.g. release-4.6).

as for putting it in the makefile or here.....  it appears to be a default here, that could be overridden from the makefile (or other invoker of the script)?

Reply:
i'll update it so we could override it in here